packages/api-server/Dockerfile-development

```diff
@@ -0,0 +1,7 @@
+FROM node:20
+
+WORKDIR /app
+
+EXPOSE 3000
+
+CMD sh -c 'npm install && node download-models.js && node index.js'
```

Comment:
Also here: `npm install` and `node download-models.js` should be moved to a RUN step to improve container startup time.

Reply:
Currently the dev container of api-server has no difference to the docker image of production. So I think we can skip this for the beginning.